composables/useChainRedirect.ts

```diff
@@ -61,6 +61,16 @@ function getRedirectPathForPrefix({
     }
   }
 
+  if (route.name === 'prefix-explore-items') {
+    const { collections: _c, page: _p, ...restOfQuery } = route.query
```

Comment:
not used?

Reply:
hi
point is to keep all query params except collections and page

eslint was complaining that the vars are unused

and it is common practice to signal unused vars by `_` prefix


could also be done by something like 
```typescript
  const restOfQuery = Object.fromEntries(
    Object.entries(route.query).filter(
      ([key]) => key !== 'collections' && key !== 'page'
    )
  )
  ```
  
  but that's ugly and less readable imo

Reply:
> and it is common practice to signal unused vars by _ prefix

`_` prefix means private var for me 

I don't understand this part of code, where `collections: _c, page: _p` will be used in this composable?

Reply:
I see now you want to delete `page` & `collections` key in `route.query`
on the first reading I wasn't aware of that, you could have use [delete](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/delete) operator

Reply:
>you could have use [delete](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/delete) operator

i thought we don't do that here